### ISSUE TITLE
Settings checkbox should left align

### ISSUE THREAD

**Issue by nlrb** [athombv/homey#844]
**What is your Homey version (Settings → System)?**

0.9.3

**What did you try to achieve?**

Add a checkbox in the driver settings.

**What was the result?**

Checkbox is centered.
![image](https://cloud.githubusercontent.com/assets/11725674/18198185/7d89446a-70fb-11e6-80fa-99cffa791d39.png)

**What did you expect as result?**

Checkbox left aligned (as in older firmwares). This looks silly.

**Any other remarks**

Tested on Chrome on Windows.
Don't know where it exactly changed, somewhere around 0.9 I guess.

**Comment by robertraaijmakers** [comment]
Tried to find a solution, this is all I could find...

The only way I found to fix this is by wrapping the checkbox in a div and removing the width: 100% from the checkbox itself (tested on Chrome for windows).

```
<div ng-if="item.type == 'checkbox'" class="input ng-scope">
	<label class="ng-binding">Tamper cancellation report</label>
	<div style="width: 100%"> <!-- add this div with a width of 100% -->
		<input class="control ng-pristine ng-valid ng-isolate-scope ng-touched" type="checkbox" name="tamper_cancellation" ng-model="item.value" dyn-attr="item.attr">
	</div> <!-- close div -->
	<div class="hint ng-scope" ng-if="item.hint" ng-include="'hintTmpl'" ng-init="hint = item.hint"><i class="hint-icon fa fa-info-circle ng-scope"></i>
		<div class="hint-popup-wrap ng-scope">
			<div class="hint-popup ng-binding">Must the tamper alarm turn off automatically?</div>
		</div>
	</div><!-- end ngIf: item.hint -->
</div>
```

```
.input>.control {
    width: 100%; /* remove this css rule */
}
```

**Comment by WeeJeWel** [comment]
What's your app.json? Because:

![screen shot 2016-10-11 at 11 54 20](https://cloud.githubusercontent.com/assets/319873/19266351/77847914-8fa9-11e6-8391-a321db876548.png)

**Comment by robertraaijmakers** [comment]
Quite interesting... Working on Chrome (Version 53.0.2785.143 m) on Windows here, maybe that has to do something with it?

The app.json is from https://github.com/athombv/com.fibaro/blob/master/app.json (line 1579) but all checkboxes I could find are this way.

```
{
					"id": "tamper_cancellation",
					"type": "checkbox",
					"label": {
						"en": "Tamper Cancellation",
						"nl": "Sabotage Annulering"
					},
					"hint": {
						"en": "Choose if the tamper alarm turns off",
						"nl": "Kies of het sabotage alarm uit gaat"
					},
					"value": true
				},
```

**Comment by WeeJeWel** [comment]
Haha yep, that only happened on Windows. Thanks! Fixed in 0.10.4.
